classes/migration/upgrade/v3_4_0/I7191_SubmissionChecklistMigration.php

```diff
@@ -49,16 +49,16 @@ public function up(): void
                         ->delete();
                     return;
                 }
-                $checklist = json_decode($row->setting_value);
+                $checklist = json_decode($row->setting_value, true);
```

Comment:
It looks correct to me 👌

FYI, I had added some fixes for it here as well:
https://github.com/pkp/pkp-lib/blob/a1dabe01c9c52291ea63b3c500792ad731f3b82c/classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php#L293-L305

Reply:
Thanks; I'm tempted not to disturb that earlier code, since it's already released and won't do harm in this case.